backends/tc/backend.cpp

```diff
@@ -191,6 +199,8 @@ void ConvertToBackendIR::setPipelineName() {
     pipelineName = pipelineName.trim();
 }
 
+cstring ConvertToBackendIR::getPipelineName() { return pipelineName; }
```

Comment:
In general, small functions like this which is only returning a member can be added to class declaration itself

Reply:
resolved